modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFieldResourceImpl.java

```diff
@@ -144,6 +146,14 @@ public ObjectField postObjectDefinitionObjectField(
 			Long objectDefinitionId, ObjectField objectField)
 		throws Exception {
 
+		if (!GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-164948")) &&
```

Comment:
Hey @seltong,
Another class that we are allowed to create object fields: ObjectDefinitionResourceImpl.java > postObjectDefinition();
Maybe we could check in ObjectFieldUtil.java..

Let me know what you think!